R/user_guide.R

```diff
@@ -281,60 +285,229 @@ render_setup_instructions <- function(path, pr) {
       server_port <- 443
   }
   apiSpec <- pr$getApiSpec()
-  desc <- markdown::markdownToHTML(text = strip_md_links(apiSpec$info$description),
+  desc <- ""
+  if (rlang::is_null(warnings)) {
+    desc <- markdown::markdownToHTML(text = strip_md_links(apiSpec$info$description),
                                    fragment.only = TRUE)
+  }
+  title <- apiSpec$info$title
+  version <- apiSpec$info$version
+  title_desc <- "title_desc not generated"
+  body_content <- "body_content not generated"
 
-  ui <- htmltools::tagList(
-    tags$header(
-      tags$h1(
-        "Tableau Setup"
-      ),
-      tags$div(class = "api-desc",
-               htmltools::HTML(desc),
-               tags$p(
-                 tags$a(
-                   href = "./",
-                   "Tableau Usage Instructions"
-                 ),
-                 tags$br(),
-                 tags$a(
-                   href = "./__docs__/",
-                   "Open API Documentation"
-                 )
-               )
+  title_desc <- htmltools::tagList(
+    tags$h1(
+      class="padded-fully title",
+      title,
+      if (!is.null(version)) paste0("(v", version, ")")
+    ),
+    tags$div(class = "api-desc",
+      tags$div(
+        class="padded-flat-top",
+        htmltools::HTML(desc)
       )
+    )
+  )
+
+  body_content <- htmltools::tagList(
+    tags$h3(
+      class="subtitle",
+      "Configure Tableau to access your extension"
```

Comment:
```suggestion
      "Configure Tableau to access this extension"
```
The more I think about it, the more "this" implies that the page might be tailored to _this_ one.

Reply:
I can't commit your change, but I will make the change myself that you recommended.